modules/quantcastIdSystem.js

```diff
@@ -34,9 +192,28 @@ export const quantcastIdSubmodule = {
    * @function
    * @returns {{id: {quantcastId: string} | undefined}}}
    */
-  getId() {
+  getId(config) {
     // Consent signals are currently checked on the server side.
     let fpa = storage.getCookie(QUANTCAST_FPA);
+
+    const coppa = coppaDataHandler.getCoppa();
+    if (coppa) {
+      logInfo('QuantcastId: IDs not provided for coppa requests, exiting QuantcastId');
+      return;
```

Comment:
why is this just `return`; whereas the case on line 40 returns `{ id: undefined }` ?